samples/dashboard/demos/climate/demo.js

```diff
@@ -144,32 +153,32 @@ async function buildDashboard() {
                         }
                     },
                     marker: {
-                        lineColor: '#FFF',
-                        lineWidth: 1,
+                        enabled: true,
                         radius: 6,
+                        // symbol: 'mapmarker'
```

Comment:
Remove?

Reply:
Not yet, mapmarker needs to be fixed in separate task.